run-local.sh

```diff
@@ -24,12 +24,14 @@ if [[ -z "$CI" ]]; then
   ./web-api/start-dynamo.sh &
   DYNAMO_PID=$!
 
-  echo "Stopping elasticsearch in case it's already running"
-  pkill -f elasticsearch
+  echo "Stopping opensearch in case it's already running"
+  $DOCKER_COMPOSE -f "$(pwd)/web-api/elasticsearch/docker-compose.yml down" --volumes || true
 
-  echo "Starting elasticsearch"
-  ./web-api/start-elasticsearch.sh &
-  ESEARCH_PID=$!
+  echo "Starting opensearch"
+  $DOCKER_COMPOSE -f "$(pwd)/web-api/elasticsearch/docker-compose.yml up" -d || { echo "Failed to start OpenSearch containers"; exit 1; }
+  
+  echo "OpenSearch is running"
+    ESEARCH_PID=$!
```

Comment:
I think this will be the process id of the previous call to `docker compose`, which will have completed by this point. It's not the process id of the running docker container.

And even if it were...